apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -57,6 +57,19 @@
 
 IS_TEMPLATE = not (API_SERVER_DIR / MODEL_LIBRARY_FORMAT_RELPATH).exists()
 
+# Maps a short, identifying microtvm device string to (target, zephyr_board).
+MICRO_DEVICES = {
+    "qemu_x86": ("host", "qemu_x86"),
```

Comment:
OK for me to move  it into the server side. Same comment applies here as for the arduino devices above.